test_scripts/ATF_ShutDown_MASTER_RESET.lua

```diff
@@ -0,0 +1,147 @@
+--  Requirement summary:
+--  Check that SDL finish its work properly MASTER_RESET with one registered App
+--
+--  Description:
```

Comment:
@LitvinenkoIra, points:
1.  SDL and HMI are running
2. App1 is registered
should be in section: "-- 1. Used preconditions(if applicable)"

For section: 
"-- 2. Performed steps" you should have actions:
Perform Master Reset
=>  HMI sends OnExitAllApplications with reason MASTER_RESET

section: "-- Expected result:" should have:
1. SDL clear all Apps folder, app_info.dat file and shut down